carbon-feature-plugin/src/main/java/org/wso2/maven/p2/utils/P2ApplicationLaunchManager.java

```diff
@@ -111,8 +111,8 @@ public void addArgumentsToUnInstallFeatures(String uninstallIUs, String destinat
                 // the location of where the plug-ins and features will be stored. This value is only taken into account
                 // when a new profile is created. For an application where all the bundles are located into the
                 // plugins/ folder of the destination, set it to <destination>
-                // use a shared location for the install. The path defaults to ${user.home}/.p2
-                "-shared", Paths.get(destination + File.separator + "p2").toString(),
+                // to support shared installation in carbon
+                "-shared", Paths.get(destination + File.separator + "lib" + File.separator + "p2").toString(),
```

Comment:
@Niranjan-K fixed with @laki88 suggestion and tested twice on both Linux and Windows